app/client/src/sagas/ActionExecution/PluginActionSaga/onPageLoadSaga.ts

```
@@ -0,0 +1,397 @@
+        },
+      },
+    );
+
+    return;
+  }
+
+  const jsAction = collection.actions.find(
+    (action: JSAction) => action.id === pageAction.id,
+  );
+
+  if (!!jsAction) {
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)